duckduckgo/macos-browser
# Tab Loading Improvements

Task/Issue URL: https://app.asana.com/0/inbox/1199237043628108/1199908097061380/1199983446696278
Tech Design URL: https://app.asana.com/0/inbox/1199237043628108/1199918950957534/1199983446696278
CC: @brindy 

**Description**:
This PR contains performance improvements for the Browser launch time especially during Session Restoration.

**Steps to test this PR**:
1. Test the Browsing Session is restored upon launch and links, context menus are working
2. Tab open in background should start loading instantly
3. New tabs should be navigatable
3. Weird yahoo.com banner problem (the banner appears sometimes when the session is restored)


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
